src/Microsoft.FeatureManagement/FeatureManager.cs

```diff
@@ -141,6 +141,13 @@ private async Task<bool> IsEnabledAsync<TContext>(string feature, TContext appCo
                     }
                 }
             }
+            else
+            {
+                if (_options.OnMissingFeature != null)
+                {
+                    return await _options.OnMissingFeature(feature).ConfigureAwait(false);
```

Comment:
I would assign the value to `enabled` instead of returning directly from here. Short-circuiting here also skipped the session manager code below.

Reply:
Oh yeah, completely missed that. Thanks.